plugins/test/unit/plugins/db/test_models.py

```diff
@@ -1133,14 +1133,15 @@ def test_render_primary(self):
         self.assertTrue('abc123' in ret)
         self.assertTrue('sha1' in ret)
 
-    def test__escape_django_template_vars(self):
+    def test__escape_django_syntax_chars(self):
         """
-        Test that a requested element is wrapped into `verbatim` templatetag.
+        Test that for a requested element all syntax characters are substituted with
+        the corresponding templatetag.
```

Comment:
What syntax characters, in particular, need to be tested? I think this information needs to be in the docstring so it's clear what should be tested, and by extension what the units under test should be doing.